torch/nn/modules/pixelshuffle.py

```diff
@@ -50,3 +54,51 @@ def forward(self, input: Tensor) -> Tensor:
 
     def extra_repr(self) -> str:
         return 'upscale_factor={}'.format(self.upscale_factor)
+
+
+class PixelUnshuffle(Module):
+    r"""Reverses the :class:`~torch.nn.PixelShuffle` operation by rearranging elements in a tensor of shape :math:`(*, C, H \times r, W \times r)` to a tensor of shape :math:`(*, C \times r^2, H, W)`, where r is a downscale factor.
```

Comment:
same here.
This might cause lint problem.
max number of character per line is 120.